Add information regarding the MVS server API to the docs (#701)
rl-institut/multi-vector-simulator: docs/Overview.rst

* Comparison to other model

Access MVS Server API
---------------------

> The multi-vector simulator can be used to simulate energy systems directly via the browser using the API of the server which processes the simulation. Users can interact with the API at this `URL <https://mvs-eland.rl-institut.de/>`_.

Reviewer: ```suggestion
The Multi-Vector Simulator can be used to simulate energy systems via the browser using the `API on the RLI server <https://mvs-eland.rl-institut.de/>`__. It processes the parsed JSON input file, and returns the JSON result file.
```
Somehow there need to be two underscores, otherwise there is a RTD warning.

> It is as simple as uploading the simulation files (by clicking the 'Browse' button) and then hitting the 'Run simulation' button. One can also visualize the log messages (error/warning) during the simulation.

Reviewer: ```suggestion
The simulation is executed by uploading the simulation files (by clicking the 'Browse' button) and then hitting the 'Run simulation' button. One can also visualize the log messages (error/warning) during the simulation.
```

> The code for the implementation of the MVS E-Land API is hosted in this `github repository <https://github.com/rl-institut/mvs_eland_api>`_.

Reviewer: ```suggestion
The code for the implementation of the MVS E-Land API is hosted in this `github repository <https://github.com/rl-institut/mvs_eland_api>`__.
```
Two undescores
